styles/dev.to/catppuccin.user.less

```
@@ -0,0 +1,275 @@
+    @text: @catppuccin[@@flavor][@text];
+    @subtext1: @catppuccin[@@flavor][@subtext1];
+    @subtext0: @catppuccin[@@flavor][@subtext0];
+    @overlay2: @catppuccin[@@flavor][@overlay2];
+    @overlay1: @catppuccin[@@flavor][@overlay1];
+    @overlay0: @catppuccin[@@flavor][@overlay0];
+    @surface2: @catppuccin[@@flavor][@surface2];
+    @surface1: @catppuccin[@@flavor][@surface1];
+    @surface0: @catppuccin[@@flavor][@surface0];
+    @base: @catppuccin[@@flavor][@base];
+    @mantle: @catppuccin[@@flavor][@mantle];
+    @crust: @catppuccin[@@flavor][@crust];
+    @accent: @catppuccin[@@flavor][@@accentColor];
+
+    color-scheme: if(@flavor = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    --body-bg: @crust;
+    --black: @crust;
+    --body-color: @text;
+
+    --header-bg: @base;
+    --header-shadow: @crust;
+
+    --footer-bg: @crust;
+    --footer-color: @subtext0;
+
+    --form-bg: @crust;
+    --form-bg-focus: @base;
+    --form-border: @surface0;
+    --form-border-hover: @surface1;
+    --form-border-focus: @accent;
+    --focus: @accent !important;
+
+    --base-100: @text;
+    --base-90: @subtext1;
+    --base-80: @subtext1;
+    --base-70: @subtext0;
+    --base-60: @overlay2;
+    --base-50: @overlay1;
+    --base-20: @base;
+    --base-0: @crust;
+    --base-inverted: @surface0;
+
+    --tab-color-current: @accent;
+    --tab-color: @text;
+    --tab-color-hover: @base;
+    --tab-bg-hover: fade(@accent, 70%);
+
+    --link-branded-color: @accent;
+    --link-branded-color-hover: @accent;
+    --link-color: @text;
+    --link-color-hover: @blue;
+    --link-bg-hover: @mantle;
+    --link-color-secondary: @subtext1;
+    --link-color-secondary-hover: @text;
+    --link-bg-current: @mantle;
+    --link-color-current: @accent;
+    --link-current-bg: @surface0;
+    --link-current-color: @accent;
+
+    --btn-bg-hover: @surface0;
+    --btn-color: @text;
+    --btn-color-hover: @accent;
+    --btn-primary-bg: @accent;
+    --btn-primary-bg-hover: fade(@accent, 90%);
+    --btn-primary-color: @base;
+    --btn-primary-color-hover: @crust;
+
+    --button-primary-color: @base;
+    --button-primary-color-hover: @base;
+    --button-primary-bg: @accent;
+    --button-primary-bg-hover: fade(@accent, 90%);
+    --button-secondary-bg: @surface1;
+    --button-secondary-bg-hover: fade(@surface1, 90%);
+    --button-secondary-color: @subtext1;
+    --button-secondary-color-hover: @text;
+    --button-outlined-bg-hover: @mantle;
+    --button-outlined-color: @subtext1;
+    --button-outlined-color-hover: @text;
+    --button-outlined-border: @surface0;
+    --button-outlined-border-hover: @surface1;
+    --button-ghost-bg-hover: @base;
+    --button-ghost-color: @text;
+    --button-ghost-color-hover: @text;
+
+    --reaction-like-bg: fade(@red, 10%);
```

I'd argue we could use accent here for the like color? What do you think?
```suggestion
    --reaction-like-bg: @accent;
```

(Was the fade necessary too?)

---

FYI, everywhere else they are using a heart emoji as reaction.

---

We usually make hearts/liking accent colored. And we can change the foreground heart icon color via https://github.com/catppuccin/userstyles/blob/main/docs/guide/images-and-svgs.md.

---

Sorry for not mentioning, it's in the comment thread of post page, [check here](https://dev.to/peledzohar/comment/2ck44).

---

Fixed.